RHDEVDOCS-3946 Pipelines 1.7 Tutorial + RHDEVDOCS-3977 Delete optional CRs before uninstalling Operator (#44804)
openshift/openshift-docs: modules/op-deleting-the-pipelines-component-and-custom-resources.adoc

====

. Click *Delete* to confirm the deletion of the CRs.

[IMPORTANT]
====

> Repeat the procedure to find and remove CRs of optional components such as {tekton-chains}, before uninstalling the Operator. If you uninstall the Operator without removing the CRs of optional components, you can not remove them later.

Reviewer: ```suggestion
Repeat the procedure to find and remove CRs of optional components such as {tekton-chains}, before uninstalling the Operator. If you uninstall the Operator without removing the CRs of optional components, you cannot remove them later.
```
